packages/web-components/fast-foundation/src/data-grid/stories/data-grid-row.register.ts

```diff
@@ -2,8 +2,9 @@ import { css } from "@microsoft/fast-element";
 import { FASTDataGridRow } from "../data-grid-row.js";
 import { dataGridRowTemplate } from "../data-grid-row.template.js";
 
-const styles = css`
+const styles = css<FASTDataGridRow>`
     :host {
+        grid-template-columns: ${x => x.dataGridContext.gridTemplateColumns};
```

Comment:
Is there a way to package up css bindings with the component/template and not depend on authors adding it in their custom css?  Seems fragile.

Reply:
You could add the css through the component's fast controller. I believe it has a an addStyles method that takes an ElementStyles instance.